cmd/influxd/server_integration_test.go

```diff
@@ -619,6 +619,31 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 		},
 		{
 			reset: true,
+			name:  "median - even sample size",
+			write: `{"database" : "%DB%", "retentionPolicy" : "%RP%", "points": [
+				{"name": "cpu", "timestamp": "2000-01-01T00:00:00Z", "tags": {"region": "us-east"}, "fields": {"value": 200}},
+				{"name": "cpu", "timestamp": "2000-01-01T00:00:10Z", "tags": {"region": "us-east"}, "fields": {"value": 30}},
+				{"name": "cpu", "timestamp": "2000-01-01T00:00:20Z", "tags": {"region": "us-east"}, "fields": {"value": 40}},
+				{"name": "cpu", "timestamp": "2000-01-01T00:00:30Z", "tags": {"region": "us-west"}, "fields": {"value": 100}}
+			]}`,
+			query:    `SELECT median(value) FROM cpu`,
+			queryDb:  "%DB%",
+			expected: `{"results":[{"series":[{"name":"cpu","columns":["time","median"],"values":[["1970-01-01T00:00:00Z",70]]}]}]}`,
+		},
+		{
+			reset: true,
```

Comment:
Every time we reset the tests, it slows stuff down. I am trying to discourage this across all our PRs. How about using a unique series name, instead of "cpu", and then selecting against that? Make sense? Then the constant "resets" won't be needed.